packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -84,9 +86,15 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
     </RunItButton>
   )
 
+  document.getElementById('top')?.scrollTo(scrollPx, 0)
```

Comment:
Open to recommendations on where to refactor this into

Reply:
make a util component for it. `<ScrollToTop>` possibly?

Reply:
We are using a package for this https://www.npmjs.com/package/react-router-hash-link and I'm hoping we can avoid having to do it manually. Is there anything in particular that makes it break on some methods?